c2corg_ui/templates/waypoint/edit.html

```diff
@@ -7,139 +9,774 @@
+
+          <div class="content collapse in" id="edit-transports">
+
+            ## PUBLIC TRANSPORT TYPES
+            <div class="form-group data half">
+              <label translate>public_transportation_types</label>
+              <ul class="types">
+                <li ng-repeat="type in ${public_transportation_types}"  ng-click="editCtrl.pushToArray(waypoint, 'public_transportation_types', type, $event)">
+                  <div class="checkbox">
+                    <input type="checkbox" ng-checked="waypoint.public_transportation_types.indexOf(type) > - 1"><span>{{type| translate}}</span>
+                  </div>
+                </li>
+              </ul>
+            </div>
+
+            ## PUBLIC TRANSPORT RATINGS
+            <div class="form-group data half">
+              <label translate>public_transportation_ratings</label>
+              <div class="input-group">
+                <select class="form-control" ng-options="rat as mainCtrl.translate(rat) for rat in ${public_transportation_ratings}" ng-model="waypoint.public_transportation_rating"></select>
+                <span class="input-group-addon"><span class="glyphicon glyphicon-star"></span></span>
+              </div>
+              <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="waypoint.public_transportation_rating"></span>
+            </div>
+
+            ## SNOW CLEARANCE RATINGS
+            <div class="form-group data half">
+              <label translate>snow_clearance_ratings</label>
+              <div class="input-group">
+                <select class="form-control" ng-options="rat as mainCtrl.translate(rat) for rat in ${snow_clearance_ratings}" ng-model="waypoint.snow_clearance_rating"></select>
+                <span class="input-group-addon"><span class="glyphicon glyphicon-star"></span></span>
+              </div>
+              <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="waypoint.snow_clearance_rating"></span>
+            </div>
+
+            ## LIFT ACCESS
+            <div class="form-group data half">
+              <label translate>lift_access</label>
+              <div class="input-group">
+                <select class="form-control" ng-model="waypoint.lift_access">
+                  <option value="true"><span translate>yes</span></option>
+                  <option value="false"><span translate>no</span></option>
+                </select>
+                <span class="input-group-addon"><span class="glyphicon glyphicon-resize-vertical"></span></span>
+              </div>
+              <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="waypoint.lift_access"></span>
+            </div>
+
+            ## PARKING FEE
+            <div class="form-group data half">
+              <label translate>parking_fee</label>
+              <div class="input-group">
+                <input type="number" name="parking_fee" ng-model="waypoint.parking_fee" class="form-control" min="1"/>
```

Comment:
This field is not a numeric field (with CHF or EUR). It's a dropdown
https://github.com/c2corg/v6_api/blob/master/c2corg_api/models/waypoint.py#L135
whose values are:
https://github.com/c2corg/v6_common/blob/master/c2corg_common/attributes.py#L195-L199